ckan/views/dataset.py

```diff
@@ -1026,116 +1061,55 @@ def activity(package_type, id):
     # TODO: remove
     g.pkg_dict = pkg_dict
     g.pkg = pkg
-    g.package_activity_stream = package_activity_stream
```

Comment:
Normally we don't fiddle with context variables and globals because even this style is deprecated, we don't want it to break someone's customized template. However the `package/activity.html` template is fundamentally changed in this PR (`package_activity_stream` changes type from pre-rendered HTML to a list), necessitating changes to a customized template anyway.